Casks/crystalmaker.rb

```diff
@@ -1,13 +1,17 @@
 cask "crystalmaker" do
-  version "10.6.0,46910"
+  version "10.6.0"
   sha256 :no_check
 
   url "http://crystalmaker.com/downloads/crystalmaker_mac.zip"
-  appcast "http://crystalmaker.com/support/updates/VersionData-CMM.xml"
   name "CrystalMaker"
   desc "Visualize crystal and molecular structures"
   homepage "http://crystalmaker.com/index.html"
 
+  livecheck do
+    url "http://crystalmaker.com/support/updates/VersionData-CMM.xml"
+    regex(/<version number="(\d+(?:\.\d+)*)">/i)
```

Comment:
```suggestion
    regex(/<version[^>]+?number=["']?v?(\d+(?:\.\d+)+)["']?[^>]*?>/i)
```

This would continue to match if the XML changes to include additional attributes in `version` elements before and/or after the `number` attribute.

It also works in situations where the attribute is enclosed by double quotes, single quotes, or no quotes at all. We'll likely never see anything other than double quotes in this particular source but I've seen all sorts of nonsense while working on livecheck, so I try not to assume.